src/sails_forth/spec.clj

```diff
@@ -137,6 +143,42 @@
                    ::precision
                    ::label
                    ::relationshipName]))
+(defmethod field-description-type "url" [_] 
+  (s/keys :req-un [::type
+                   ::picklistValues
+                   ::name
+                   ::referenceTo
+                   ::scale
+                   ::precision
+                   ::label
+                   ::relationshipName]))
+(defmethod field-description-type "multipicklist" [_] 
+  (s/keys :req-un [::type
+                   ::picklistValues
+                   ::name
+                   ::referenceTo
+                   ::scale
+                   ::precision
+                   ::label
+                   ::relationshipName]))
+(defmethod field-description-type "address" [_] 
+  (s/keys :req-un [::type
+                   ::picklistValues
+                   ::name
+                   ::referenceTo
+                   ::scale
+                   ::precision
+                   ::label
+                   ::relationshipName]))
+(defmethod field-description-type "phone" [_] 
+  (s/keys :req-un [::type
+                   ::picklistValues
+                   ::name
+                   ::referenceTo
+                   ::scale
+                   ::precision
+                   ::label
+                   ::relationshipName]))
```

Comment:
Good, well-chosen subset. We could probably go farther for some of these types and be prescriptive about the expected values also, but this is super fine for this iteration.